config/src/config/storage_config.rs

```diff
@@ -118,6 +120,12 @@ pub struct StoragePrunerConfig {
     pub state_store_pruning_batch_size: usize,
     /// The offset for user pruning window to adjust
     pub user_pruning_window_offset: u64,
+    /// The pruning interval of state pruner. The state pruner will sleep for this period of time
+    /// after pruning each batch.
+    pub state_pruner_time_interval_in_ms: u64,
```

Comment:
comment not accurate and deserves better name.

Actually, I don't think this needs to be configurable. let's just do 100ms, for e.g., and make it an constant. Thoughts, @grao1991 ?

Reply:
I made it configurable because otherwise the tests take forever to run.

Reply:
use a `if cfg!(test)`?

Reply:
Make it non-configurable.